app/docs/2.0.x/deployments/multi-zone.md

```diff
@@ -23,6 +23,18 @@ This way you can make a service multi-zone by having data planes using the same
 Let's look at how a service `backend` in `zone-b` is advertised to `zone-a` and a request from the local zone `zone-a` is routed to the remote
 service in `zone-b`.
 
+```mermaid
+graph TD
+    subgraph zone-A
+        client[Client]
+        serviceA[Service Backend A]
+    end
+    subgraph zone-B
+        serviceB[Service Backend B]
+    end
+    client --> serviceB
+```
```

Comment:
```suggestion
{% mermaid %}
graph TD
    subgraph zone-A
        client[Client]
        serviceA[Service Backend A]
    end
    subgraph zone-B
        serviceB[Service Backend B]
    end
    client --> serviceB
{% endmermaid %}
```

Reply:
OK, got it Bart. Thank you.

I just changed it and force pushed to avoid multiple commits.